### DISCUSSION TITLE
作者大大我有一个问题

### DISCUSSION THREAD

**Discussion by Athena233** [RapidAI/TableStructureRec#101] [Q&A]
![2021 acl-long 19 pdf-3-0](https://github.com/user-attachments/assets/d7b9b48d-da68-492f-a1f5-1a862c4da6bb)
<img width="611" alt="06f36f8505160ce0918f4cf466195ec" src="https://github.com/user-attachments/assets/d23d0171-3386-41e2-9be5-edd55d98d226" />
<img width="637" alt="d7aaa83ce4500ccb30e42383f7aba68" src="https://github.com/user-attachments/assets/2fe383e6-8fea-4448-a104-fbada179b040" />
我在线测试的时候，发现有不少这种无线表格识别的不太好，是模型的原因吗

**Comment by Joker1212** [comment]
是，无线表格，它就是不行，根源在模型本身

**Reply by Athena233** [reply]
那对于这种无线表、三线表你有什么好的办法嘛，我自己的数据集就是这种数据偏多

**Reply by Joker1212** [reply]
可以基于读光的模型进行微调训练哈，还有一个难搞的就是复现论文了，感觉seg and merge 两步的做法不错，可惜最近的都不开源

**Reply by Sherryran08** [reply]
想问一下哥，你说的这个复现论文的seg and merge 两步的做法是指的哪篇论文？最近也在尝试无线表格性能优化这块工作，有在SLANet plus上微调无线表格的识别能力，但是感觉效果不佳

**Reply by Joker1212** [reply]
目前有三种技术方向哈，rapidTable是序列预测的方式，wiredV2是识别横线竖线再分割得到单元格的sam方式，我个人倾向于split andd merge的方向，论文可以参考  [TSRFormer](https://arxiv.org/abs/2203.01017), [TSRFormer DQ-DETR](https://arxiv.org/abs/2303.11615), 序列预测的话，感觉 [DRCC](https://www.ijcai.org/proceedings/2023/152) [Unitable](https://arxiv.org/abs/2403.04822) 不错

> 想问一下哥，你说的这个复现论文的seg and merge 两步的做法是指的哪篇论文？最近也在尝试无线表格性能优化这块工作，有在SLANet plus上微调无线表格的识别能力，但是感觉效果不佳

**Reply by Sherryran08** [reply]
感谢🙏，btw，感谢开源这么好的工作
